### ISSUE TITLE
Crash sur mon Windows 7 64Bits 

### ISSUE THREAD

**Issue by Zer01neDev** [dridk/freebox-desktop#51]
Crash après quelque seconde lors du téléchargement de gros fichiers, l'application devient de plus en plus lourde en mémoire, il faut penser a libérer la mémoire des buffer inutilisé.

**Comment by dridk** [comment]
Sur freebox torrent?

**Comment by Zer01neDev** [comment]
Non, lors que je télécharge un gros fichier (>1GB) de ma Freebox a mon ordinateur via freebox-desktop, a partir de 3XXMB l'application crash aucun problème pour les fichier en dessous de 1GB, et aucun problème de ce genre sous un système 32Bits..
